build.rs

```diff
@@ -10,9 +10,11 @@ fn main() {
         autocfg::emit("has_i128");
     }
 
+    let autocfg = AutoCfg::new().unwrap();
```

Comment:
I meant sharing even earlier -- on line 9, `autocfg::new()` does the same as this `AutoCfg::new().unwrap()`.

Reply:
My bad, didn't even see that… as you can guess, I'm not that familiar with autocfg